staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go

```diff
@@ -189,5 +189,17 @@ func (c *APIServiceRegistrationController) deleteAPIService(obj interface{}) {
 		}
 	}
 	klog.V(4).Infof("Deleting %q", castObj.Name)
-	c.enqueue(castObj)
+	c.enqueueInternal(castObj)
+}
+
+// Enqueue queues all apiservices to be rehandled.
```

Comment:
this conforms to an interface? You'll want explain why.

Reply:
done.